nuttx-configs/s2740vc-v1/Kconfig

```diff
@@ -3,6 +3,6 @@
 # see misc/tools/kconfig-language.txt.
 #
 
-if CONFIG_ARCH_BOARD_S2740VC
+if CONFIG_ARCH_BOARD_S2740VC_V1
```

Comment:
Well spotted David. Yes, that was the only non-cosmetic change, which may
avoid a gotcha for somebody relying on that define some day in the future.

I wasn't sure about the worth of posting these kinds of cosmetic changes. I
reduced the diff loading whilst examining the differences during a review
of all the CAN based drivers, and if there's one thing I despise more than
trivial comments that possibly add no real value, it's comments that
actually present incorrect information! I've already been bitten quite hard
by one of such instance of a non-trivial comment during my work on the 9250
driver.

I'd actually like to remove the <path-to>filename lines in the various
headers, as they're only going to be wrong again should the tree structure
shift - and I'm unaware of any value from them. Going somewhat further, I'd
like to change some of the filenames in these modules to be identical, such
as renaming 's2740vc_init.c' and 'px4esc_init.c', to be 'init.c' etc. This
way things line-up in a Meld compare, and makes it much easier to review
and learn about the code. I held back from making such changes as I thought
better to consult with the rest of team first. I'm thinking some will deem
such a waste of effort and leading to change for change sake, others may
see the value - after all it doesn't take much to pursue some presentation
perfection!

I'm suspecting MW may have reservations given the consequences of what I
did with a whitespace cleanup in the MatrixPilot codebase some years back..

Thoughts?


On 30 April 2016 at 03:22, David Sidrane <notifications@github.com> wrote:

> In nuttx-configs/s2740vc-v1/Kconfig
> <https://github.com/PX4/Firmware/pull/3131#discussion_r61614617>:
>
> > @@ -3,6 +3,6 @@
> >  # see misc/tools/kconfig-language.txt.
> >  #
> >
> > -if CONFIG_ARCH_BOARD_S2740VC
> > +if CONFIG_ARCH_BOARD_S2740VC_V1
>
> @kd0aij <https://github.com/kd0aij> - looks to be except for this change
> that is a NOP
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/PX4/Firmware/pull/3131/files/e9f5bffe714021f031f303ce9444a9f0a8a3762b..c1d66d157208d82866d0dd25a4ccf9a7b12b3946#r61614617>
>